recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bbappend

```diff
@@ -17,6 +19,9 @@ do_install_append() {
 
     install -d ${D}${systemd_unitdir}/system/
     install -m 0644 ${WORKDIR}/RteCtrl.service ${D}${systemd_unitdir}/system/
+
+    install -d ${D}{bindir}
+    install -m 0755 ${WORKDIR}/rte_ctrl.sh ${D}${bindir}
```

Comment:
```suggestion
    install -m 0755 ${S}/src/${GO_IMPORT}/scripts/rte_ctrl ${D}${bindir}
```

Reply:
Also, both of these lines should be moved to the `.bb` file, not the `.bbappend`

Reply:
bbapend should only deliver custom configuration

Reply:
Change added. Resolving